scss/tech.scss

```diff
@@ -1,39 +1,34 @@
 
 .techs-container {
-  display: flex;
-  flex-wrap: wrap;
-  margin-left: 68px;
+  display: grid;
+  grid-template-areas: 
+    "main-tech tech tech"
+    "main-tech tech tech";
```

Comment:
There's no actual need for this, you can use `grid-template-column: repeat(3, max-content);` :smile: